src/foam/nanos/crunch/crunchtest/CapabilityTest.java

```diff
@@ -397,7 +396,7 @@ public void testCapability(X x) {
     c1.setNotes("noted");
     c1.setVersion("0.0.0.0");
     c1.setEnabled(true);
-    c1.setVisible(false);
+    c1.setVisibilityPredicate(TRUE);
```

Comment:
why set this at all? - not big deal

Reply:
`setVisible` doesn't exist anymore so I had to update it. Didn't think about removing visibility... this is a good reminder that we should structure test cases so it's really easy to add more cases - we should be able to change that test case so it tries it both ways